### DISCUSSION TITLE
Error E5113 "Error while calling loa chunks", impossible to install NvChad

### DISCUSSION THREAD

**Discussion by axddn** [NvChad/NvChad#2054] [Q&A]
Hello,

So I'm running Nvim 0.9.0 in iTerm2 on Mac OS 13.4.
When I try to install NvChad just with this line : 
git clone https://github.com/NvChad/NvChad ~/.config/nvim --depth 1 && nvim
And I launch nvim for the first time, NVim can't install plugins and I have this error : 
![Screenshot 2023-05-25 at 12 31 12](https://github.com/NvChad/NvChad/assets/20702781/6a10f10a-d4cb-4f94-b8ba-4f4e02eee25e)

I tried to delete ~/.local/share/nvim and launch again nVim like I saw on internet but it's not working.

I'd like to use NvChad so if someone has an idea of what's wrong it would be great.
Thx !

**Comment by siduck** [comment]
@Unslaad  rm -rf .local/share/nvim

**Reply by axddn** [reply]
@siduck
"I tried to delete ~/.local/share/nvim and launch again nVim like I saw on internet but it's not working."

First thing I tried but it's not working

**Reply by siduck** [reply]
@Unslaad could you ask in our help channel in our discord group? i'm a little busy now, pretty sure someone would help  you there!
